### PR TITLE
Feature/civ 2693 2991

### PR DESCRIPTION
**Pull request by smitatalawadekar-hmcts**
**Before creating a pull request make sure that:**

- [X] commit messages are meaningful and follow good commit message guidelines
- [X] README and other documentation has been updated / added (if needed)
- [X] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-2991
https://tools.hmcts.net/jira/browse/CIV-2693

### Change description ###
Access profiles for GA staff

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by drummondjm — DISMISSED** [review]

**Review by vasudevganesanhmcts — APPROVED** [review]

File: mocks/wiremock/mappings/claimstore/claimantClaims.json [hunk 1]
```diff
@@ -1,405 +1,405 @@
-{
```
Comment (smitatalawadekar-hmcts) [inline mocks/wiremock/mappings/claimstore/claimantClaims.json:1]: This file only has differences in line separators.

File: mocks/wiremock/mappings/claimstore/defendantClaims.json [hunk 2]
```diff
@@ -1,405 +1,405 @@
-{
```
Comment (smitatalawadekar-hmcts) [inline mocks/wiremock/mappings/claimstore/defendantClaims.json:1]: This file only has differences in line separators.

File: mocks/wiremock/mappings/fees-register/fee-range.json [hunk 3]
```diff
@@ -1,33 +1,33 @@
-{
```
Comment (smitatalawadekar-hmcts) [inline mocks/wiremock/mappings/fees-register/fee-range.json:1]: This file only has differences in line separators.

File: mocks/wiremock/mappings/legacy-draft-store/returnDraftClaim.json [hunk 4]
```diff
@@ -1,125 +1,125 @@
-{
```
Comment (smitatalawadekar-hmcts) [inline mocks/wiremock/mappings/legacy-draft-store/returnDraftClaim.json:1]: This file only has differences in line separators.

**Pull request merged**